GiniVision/Classes/OnboardingPage.swift

```diff
@@ -84,9 +84,9 @@ import UIKit
         // Image view
         imageView.translatesAutoresizingMaskIntoConstraints = false
         ConstraintUtils.addActiveConstraint(item: imageView, attribute: .top, relatedBy: .equal, toItem: contentView, attribute: .top, multiplier: 1, constant: 0)
-        ConstraintUtils.addActiveConstraint(item: imageView, attribute: .width, relatedBy: .lessThanOrEqual, toItem: nil, attribute: .width, multiplier: 1, constant: 204)
+        ConstraintUtils.addActiveConstraint(item: imageView, attribute: .width, relatedBy: .lessThanOrEqual, toItem: nil, attribute: .width, multiplier: 1, constant: 612)
```

Comment:
612... that' quite unusual but very specific. How did you calculate it? And if it's calculated, why not calculate it dynamically?

Reply:
I didn't like to harcode the values neither. At first I created some static properties for them, do you think It would be better or it would be enough just adding a comment above explaining?